<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewRelationDialog</class>
 <widget class="QDialog" name="NewRelationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>231</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>231</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Relation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="relationName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
     <property name="placeholderText">
      <string>choose a name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>relation name</string>
     </property>
     <property name="buddy">
      <cstring>relationName</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="domain2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>domain 2</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_2</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="comboBox_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="domain1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>domain 1 </string>
        </property>
        <property name="buddy">
         <cstring>comboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" alignment="Qt::AlignRight">
       <widget class="QLabel" name="alert_cb">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="alert_cb2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" alignment="Qt::AlignRight">
    <widget class="QLabel" name="alert_relationName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>initial state</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>homogeneous</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="stateBox"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>relationName</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>NewRelationDialog</receiver>
   <slot>refreshUiState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>NewRelationDialog</receiver>
   <slot>refreshUiState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>relationName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>NewRelationDialog</receiver>
   <slot>refreshUiState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewRelationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewRelationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshUiState()</slot>
 </slots>
</ui>
